dev/buildtool/halyard_commands.py

```diff
@@ -209,7 +209,8 @@ def _do_repository(self, repository):
 
     if (os.path.isfile(os.path.join(repository.git_dir,
                                     "gradle", "init-publish.gradle"))):
-      args.append('-I gradle/init-publish.gradle')
+      args.extend(
+          ['-I gradle/init-publish.gradle', '-Prelease.useLastTag=true'])
 
```

Comment:
I think in the new gradle plugin there is a flag specifying which version to use. I use it in the debian or gradle command handler. That woudl be better that use last tag. If we no longer need use last tag then we probably wont need to delete all the tags anymore and can just specify the tag we want to use as we have been asking for.